src/main/java/org/fundacionjala/sfdc/pageobjects/SalesForceLogIn.java

```diff
@@ -0,0 +1,102 @@
+package org.fundacionjala.sfdc.pageobjects;
+
+import org.fundacionjala.sfdc.commons.DriverManager;
+import org.fundacionjala.sfdc.util.CommonActions;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.How;
+import org.openqa.selenium.support.PageFactory;
+import org.openqa.selenium.support.ui.ExpectedConditions;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.openqa.selenium.support.ui.ExpectedConditions'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)